float64 joint1
float64 joint2
float64 joint3
float64 joint4
float64 joint5
---
float64 x
float64 y
float64 z
float64 qx
float64 qy
float64 qz
float64 qw
string message
bool success

